yarn.lock

```diff
@@ -927,14 +927,13 @@
   resolved "https://registry.yarnpkg.com/@chaitanyapotti/random-id/-/random-id-1.0.3.tgz#f52f647cfe9f79fc7723ea2b01b0ad3889204002"
```

Comment:
You shouldn't be changing the `yarn.lock` if no packages where installed or removed.

Reply:
Yeah. Ofc, but when I start a project, should try `yarn install`. yarn.lock file is changed when I try to install dependencies.

Reply:
If there is change on the lock when you install it should be solved on another PR, i did some update of the lock file here: https://github.com/snapshot-labs/snapshot/pull/517

Reply:
This PR should not change the lock file